code.py

```diff
@@ -0,0 +1,11 @@
+import json
+
+
+def lambda_handler(event, context):
+    if "body" not in event:
+        return "no body found"
+
+    # 1. Parse GitHub Webhook
+    payload = json.loads(event["body"])
+    print(payload)
```

Comment:
> [!CAUTION]
> Return statement at the end of the function has the same issue - it's returning a plain string instead of a proper HTTP response structure.

> [!TIP]
> Return a properly structured HTTP response object with success status code.

```python
return {
    "statusCode": 200,
    "headers": {"Content-Type": "application/json"},
    "body": json.dumps({"message": "ok"})
}
```